### PR TITLE
upload governance.md, SIG template and SIG governance

### PR DESCRIPTION
**Pull request by gingerkidney**

### CODE REVIEW

**Comment by gingerkidney** [comment]
@dcalvin @winkyao PTAL

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Community Membership
+
+Refer to [Community Membership](community-membership.md)
+
+## Community Organization
+
+![community organization](media/governace/community_organization.png)
```
Comment (winkyao) [inline GOVERNANCE.md]: `media/governace/community_organization.png` -> `/media/governace/community_organization.png`

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Committer:  Recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A Committer must have completed at least one feature or fix a major bug independently.
+- Reviewer: Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors. Reviewer has the rights to approve code reviews with LGTM (Looks Good To Me).
+- Active contributor: Contributed 8 PRs and above to TiKV within one year and received recognition from 2 incumbent Reviewers
+- Contributor: Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
+
+Organizer Committee: The project organization committee is responsible for the implementation, export, and promotion of technical content. The members include the user group leaders of each region and the person in charge of community activities.
+
+## Developer Group Structure
+
+![developer group](media/governace/developer_group.png)
```
Comment (winkyao) [inline GOVERNANCE.md]: ditto, you need to add “/” before the path.

**Comment by winkyao** [comment]
@dcalvin Yep, I agree.

**Pull request merged**